### ISSUE TITLE
Add proper Rspec tests

### ISSUE THREAD

**Issue by ezekg** [ezekg/sass-prof#10]

**Comment by chriseppstein** [comment]
Also interesting: A test case that compiles some file and fails if some performance criteria is unmet.

**Comment by ezekg** [comment]
Is there a way to compile a string of Sass code from within Ruby (without actually creating any Sass files)? That might make those types of tests easier to manage.

**Comment by chriseppstein** [comment]
Yep. `Sass.compile` https://github.com/sass/sass/blob/stable/lib/sass.rb#L55
